owncloudApp/src/main/java/com/owncloud/android/datamodel/FileDataStorageManager.java

```diff
@@ -1190,7 +1191,7 @@ public boolean saveShare(OCShare share) {
             if (result_uri != null) {
                 long new_id = Long.parseLong(result_uri.getPathSegments()
                         .get(1));
-                share.setId(new_id);
+//                share.setId(new_id);
```

Comment:
why do you hide it? if it can be deleted, do it. Don't be afraid ;-)

Reply:
I guess I forgot it...